test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -1400,6 +1400,24 @@ TEST_F(AtenXlaTensorTest, TestDropoutInPlace) {
   ExpectCounterChanged("xla::bernoulli", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestNativeDropout) {
```

Comment:
I think we need at least 3 test cases to cover the lowering logic

- prob > 0, train = true
- prob = 0, train = true
- train = false

Reply:
@zpcore I think prob = 0 train = true test is missing. Can you please add this case? This is a special handling in the lowering logic